docs/examples/sequence_processing/optical_flow_example.ipynb

```diff
@@ -189,7 +189,7 @@
    "source": [
     "## Using Dali\n",
     "### Define the Pipeline.  \n",
-    "For advanced usage, refer to SequenceReader and VideoReader docs."
+    "For advanced usage, refer to readers.Sequence and readers.Video docs."
```

Comment:
perhaps out of scope but I'll write it here for the record. We should probably add links to the operator documentation when mentioning operators in the examples.

Reply:
Certainly out of scope. There is a lot of those mentions, it would be a really tedious task. (not like this isn't). Nonetheless good idea.